doc/source/getting_started/install.rst

```diff
@@ -262,7 +262,7 @@ Visualization
 ========================= ================== =============================================================
 Dependency                Minimum Version    Notes
 ========================= ================== =============================================================
-setuptools                38.6.0             Utils for entry points of plotting backend
+setuptools                51.0.0             Utils for entry points of plotting backend
```

Comment:
the notes suggest the min version we require  is for using the plotting backend.

Reply:
Yes, this doesn't need to be changed.